libs/coin-modules/coin-solana/src/__tests__/unit/hw-signMessage.unit.test.ts

```diff
@@ -0,0 +1,50 @@
+import { SignerContext } from "@ledgerhq/coin-framework/lib/signer";
+import { Account, AnyMessage, TypedEvmMessage } from "@ledgerhq/types-live";
+import { signMessage } from "../../hw-signMessage";
+import { SolanaSigner } from "../../signer";
+
+describe("Testing call to hardware off-chain sign message on Solana", () => {
+  it("should sign a message off-chain on hardware and return the generated signature", async () => {
+    const SIGNATURE =
+      "4gVuB1KsM58fb3vRpnDucwW4Vi6fVGA51QDQd9ARvx4GH5yYVDPzDnvzUbSJf3YLWWdsX7zCMSN9N1GMnTYwWiJf";
+
+    const signMessageMock = jest.fn((_deviceId, _messageHex) => {
+      return Promise.resolve({ signature: Buffer.from(SIGNATURE) });
+    });
+    const solanaSigner: SolanaSigner = {
+      getAddress: jest.fn(),
+      signTransaction: jest.fn(),
+      signMessage: signMessageMock,
+    };
+
+    const signerContext: SignerContext<SolanaSigner> = (_, fn) => fn(solanaSigner);
+    const accountFreshAddressPath = "44'/60'/0'/0/0";
```

Comment:
nitpick, but for consistency, this is an ethereum derivation path.

```suggestion
    const accountFreshAddressPath = "44'/501'/0'/0/0";
```

Reply:
done